pkg/tsdb/influxdb/response_parser.go

```diff
@@ -85,8 +85,10 @@ func transformRows(rows []Row, query Query) data.Frames {
 			field := data.NewField("value", nil, values)
 			frames = append(frames, data.NewFrame(row.Name, field))
 		} else {
+			timeName := "Time"
```

Comment:
I'd argue we should leave this capitalized by default since this is the value that is currently saved in the panel JSON for all customers who have dashboards created < 9.3.8?

Reply:
So I was wrong, the difference is between FE/BE so there are going to be users with both strings from potentially a few versions. I think we'll need to migrate the panel JSON to be consistent before the frontend gets it?

Reply:
Yeah, no matter what we change the future values to, the frontend is going to be confronted with both capitalizations between the value saved in the transformation and the new values from the data source. So just changing this back will only move the problem to a different set of users. We'll need to migrate the JSON for the transformation before it's sent to the web application. @kylebrandt @LudoVio I might need some help with this!